crates/common_utils/src/id_type/global_id.rs

```diff
@@ -79,6 +79,23 @@ impl CellId {
     }
 }
 
+impl<'de> serde::Deserialize<'de> for CellId {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        let deserialized_length_id = LengthId::deserialize(deserializer)?;
+        Ok(Self(deserialized_length_id))
+    }
+}
+
+impl Default for CellId {
```

Comment:
do we need the default impl?

Reply:
As the parent type requires it, we will need a default impl for this.